rewrite-maven/src/main/java/org/openrewrite/maven/tree/ResolvedPom.java

```diff
@@ -55,34 +54,18 @@ public class ResolvedPom {
     public static final PropertyPlaceholderHelper placeholderHelper = new PropertyPlaceholderHelper("${", "}", null);
 
     // https://maven.apache.org/ref/3.6.3/maven-model-builder/super-pom.html
-    private static final ResolvedPom SUPER_POM = new ResolvedPom(
-            new Pom(null, null, null, null, null, null, emptyMap(), emptyList(), emptyList(), singletonList(MavenRepository.MAVEN_CENTRAL), emptyList(), emptyList(), null, null),
-            emptyList()
-    );
+    private static final ResolvedPom SUPER_POM = ResolvedPom.builder()
+            .requested(Pom.builder()
+                    .repositories(singletonList(MavenRepository.MAVEN_CENTRAL))
+                    .build())
+            .build();
 
     @With
     Pom requested;
 
     @With
     Iterable<String> activeProfiles;
 
-    public ResolvedPom(Pom requested, Iterable<String> activeProfiles) {
-        this(requested, activeProfiles, emptyMap(), emptyList(), null, emptyList(), emptyList(), emptyList(), emptyList());
-    }
-
-    @JsonCreator
-    ResolvedPom(Pom requested, Iterable<String> activeProfiles, Map<String, String> properties, List<ResolvedManagedDependency> dependencyManagement, @Nullable List<MavenRepository> initialRepositories, List<MavenRepository> repositories, List<Dependency> requestedDependencies, List<Plugin> plugins, List<Plugin> pluginManagement) {
```

Comment:
This worries me. Where is the constructor going?

Reply:
After this PR there are no remaining invocations of the two-argument form of the constructor within [OpenRewrite](https://app.moderne.io/results/uv5uuVFm2).
So if that's gone the remaining constructor is exactly what lombok would generate, so there is no need to preserve it.